### PR TITLE
docs: building an image

### PR DESCRIPTION
**Pull request by themr0c**
docs: building an image

### CODE REVIEW

File: website/docs/getting-started/building-an-image.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+sidebar_position: 10
+title: Building an image
+description: Building an image on your Container engine.
```
Comment (benoitf) [inline website/docs/getting-started/building-an-image.md]: curious why Container is with an uppercase

File: website/docs/getting-started/building-an-image.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+keywords: [podman desktop, podman, containers, images]
+tags: [images]
+---
+
+# Building an image on your Container engine
+
+With Podman Desktop, you can build an image from a Containerfile on your Podman container engine.
```
Comment (benoitf) [inline website/docs/getting-started/building-an-image.md]:
```suggestion
With Podman Desktop, you can build an image from a Containerfile on your container engine.
```

File: website/docs/getting-started/building-an-image.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+
+#### Prerequisites
+
+* Your Containerfile: `Containerfile`.
```
Comment (benoitf) [inline website/docs/getting-started/building-an-image.md]: I would add also `Dockerfile` as most of the people have Dockerfile

File: website/docs/getting-started/building-an-image.md (hunk 1 above)
Comment (themr0c) [inline website/docs/getting-started/building-an-image.md]:
```suggestion
description: Building an image on your container engine.
```

File: website/docs/getting-started/building-an-image.md (hunk 3 above)
Comment (themr0c) [inline website/docs/getting-started/building-an-image.md]:
```suggestion
* Your Containerfile: `Containerfile` or `Dockerfile.
```

File: website/docs/getting-started/building-an-image.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+---
+
+# Building an image on your Container engine
+
+With Podman Desktop, you can build an image from a Containerfile on your container engine.
+
+
+#### Prerequisites
+
+* Your Containerfile: `Containerfile` or `Dockerfile.
```
Comment (themr0c) [inline website/docs/getting-started/building-an-image.md]:
```suggestion
* Your Containerfile: `Containerfile` or `Dockerfile`.
```

File: website/docs/getting-started/building-an-image.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+* Your Containerfile: `Containerfile` or `Dockerfile`.
+
+#### Procedure
+
+1. Open **Podman Desktop dashboard > Images**.
+1. Click **<icon icon="fa-solid fa-cube" size="lg" /> Build an image**.
+1. On the **Build Image from Containerfile** screen
+   1. **Containerfile path**: select the Containerfile to build.
+   2. **Image Name**: edit the image name: `my_image`.
```
Comment (themr0c) [inline website/docs/getting-started/building-an-image.md]:
```suggestion
   1. **Image Name**: edit the image name: `my_image`.
```

File: website/docs/getting-started/building-an-image.md (hunk 1 above)
Comment (themr0c) [reply website/docs/getting-started/building-an-image.md]: mistake

**Review by benoitf — APPROVED** [review]

**Pull request merged**